code/modules/vehicles/cm_armored.dm

```diff
@@ -380,7 +379,10 @@ var/list/TANK_HARDPOINT_OFFSETS = list(
 		W.take_damage(30)
 		var/obj/vehicle/multitile/root/cm_armored/CA = root
 		CA.take_damage_type(10, "blunt", W)
-		playsound(W, 'sound/effects/metal_crash.ogg', 35)
+		if(!cmdevsplzcanyourspam)
+			playsound(W, 'sound/effects/metal_crash.ogg', 35)
+			spawn(10)
+				cmdevsplzcanyourspam = 0
```

Comment:
Instead of using spawns for this sort of thing you want to instead record a last sound time from `world.time`
So for example it would be
```
var/lastsound
...
if(world.time > lastsound + 10)
  ...
  lastsound = world.time
```

Reply:
Sure!

Reply:
Whoops, one sec!

Reply:
@ninjanomnom Fixed hopefully!